internal/journal/table_of_contents.lua

```diff
@@ -12,45 +12,81 @@ TableOfContents = defclass(TableOfContents, widgets.Panel)
 TableOfContents.ATTRS {
     frame_style=INVISIBLE_FRAME,
     frame_background = gui.CLEAR_PEN,
-    on_submit=DEFAULT_NIL
+    on_submit=DEFAULT_NIL,
+    text_cursor=DEFAULT_NIL
 }
 
 function TableOfContents:init()
     self:addviews({
         widgets.List{
-            frame={l=1, t=0, r=1, b=0},
+            frame={l=0, t=0, r=0, b=3},
             view_id='table_of_contents',
             choices={},
             on_submit=self.on_submit
         },
+        widgets.HotkeyLabel{
+            frame={b=0},
+            key='A_MOVE_N_DOWN',
```

Comment:
you'll have to gate this on the DF version as well. the keys listed here must be valid for the DF version in order for the UI to load

Reply:
fixed